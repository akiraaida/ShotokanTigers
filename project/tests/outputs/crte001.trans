10 admin                00000 00000.00 A 
05 12345678901234567890 00000 00100.00   
00                      00000 00000.00   
