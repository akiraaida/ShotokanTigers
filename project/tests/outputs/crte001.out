Please enter your session type: 
Please enter a command admin: 
Please enter the new account holder's name: 
Please enter their initial balance: 
Success, account created.